### PR TITLE
Inspector buttons

### PR DESCRIPTION
**Pull request by huwshimi**
Add the delete button and confirmation to the service overview.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- visit http://0.0.0.0:6543/:flags:/react/?deploy-target=cs%3Atrusty%2Fwordpress
- click on the wordpress service once it appears
- there should now be a Destroy button at the bottom of the inspector
- click Destroy
- the confirmation should slide up from the bottom
- click cancel
- the confirmation should slide out of view

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2937/
Test PASSed.

**Comment by urosj** [comment]
QA OK! Nice work!

File: jujugui/static/gui/src/app/components/service-overview/test-service-overview.js [hunk 1]
```diff
@@ -61,24 +70,21 @@ describe('ServiceOverview', function() {
           }
         };
       }};
-    var shallowRenderer = testUtils.createRenderer();
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/service-overview/test-service-overview.js]: nice that wrapper cleans it up quite a bit.

File: jujugui/static/gui/src/app/components/service-overview/service-overview.js [hunk 2]
```diff
@@ -181,18 +183,59 @@ YUI.add('service-overview', function() {
       this.state.actions = actions;
     },
 
+    /**
+      Set the confirmation state to open.
+      @method _showConfirmation
+    */
+    _showConfirmation: function() {
+      this.setState({confirmationOpen: true});
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/service-overview/service-overview.js]: nice

**Comment by hatched** [comment]
:+1: !

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1030

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**